include/localization.js

```diff
@@ -36,9 +33,7 @@ module.exports = function LocalizationModule(pb) {
      * @param {Object} request The request object
      */
     function Localization(request){
-        
-        //expected to be lowercase and of the form "en-us"
-        this.language = Localization.best(request).toString();
+        this.language = Localization.best(request).toString().toLowerCase().replace('-', '_');
```

Comment:
I don't think the ```.toLowerCase``` and ```.replace``` are necessary anymore after the changes for allowing users to set their locale preference.